### PR TITLE
Update lookup controllers

### PR DESCRIPTION
**Pull request by HauklandJ**
## Description
Updates the lookup controllers to ahdere to the standard set by the telemetry middleware. Expose http codes semantically. Log errors.

## Related Issue(s)
- #{issue number}

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/Altinn.App.Api/Controllers/LookupOrganisationController.cs [hunk 1]
```diff
@@ -43,11 +44,39 @@ ILogger<LookupOrganisationController> logger
...
+    private async Task<ServiceResult<Organization?, ProblemDetails>> GetOrganisationDataOrError(string orgNr)
+    {
+        Organization? organisation;
+        try
+        {
+            organisation = await _organisationClient.GetOrganization(orgNr);
+        }
+        catch
+        {
+            return new ProblemDetails
```
Comment (martinothamar) [inline src/Altinn.App.Api/Controllers/LookupOrganisationController.cs:72]: Should the exception be logged somewhere?

**Review by martinothamar — APPROVED** [review]

File: src/Altinn.App.Api/Controllers/LookupOrganisationController.cs (hunk 1 above)
Comment (HauklandJ) [reply src/Altinn.App.Api/Controllers/LookupOrganisationController.cs:72]: log and trace coming up

File: test/Altinn.App.Api.Tests/OpenApi/swagger.yaml [hunk 2]
```diff
@@ -1576,6 +1588,24 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/LookupPersonResponse'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetails'
+        '403':
```
Comment (cammiida) [inline test/Altinn.App.Api.Tests/OpenApi/swagger.yaml:1597]: Skulle 429 vært med her? 😄

File: src/Altinn.App.Api/Controllers/LookupOrganisationController.cs [hunk 3]
```diff
@@ -43,11 +44,40 @@ ILogger<LookupOrganisationController> logger
...
+
+        return Ok(LookupOrganisationResponse.CreateFromOrganisation(organisationResult.Ok));
+    }
+
+    private async Task<ServiceResult<Organization?, ProblemDetails>> GetOrganisationDataOrError(string orgNr)
+    {
+        Organization? organisation;
+        try
+        {
+            organisation = await _organisationClient.GetOrganization(orgNr);
```
Comment (cammiida) [inline src/Altinn.App.Api/Controllers/LookupOrganisationController.cs]: Kunne det vært nice å bare returnere inni `try`?

**Review by cammiida — APPROVED** [review]
Very nice 🙌

File: test/Altinn.App.Api.Tests/OpenApi/swagger.yaml (hunk 2 above)
Comment (HauklandJ) [reply test/Altinn.App.Api.Tests/OpenApi/swagger.yaml:1597]: swaggeren blir autogenerert, burde plukket det opp. Manuell endring overskriver. Ivar ser på swagger gen nå, så jeg dypper ikke ned i det

**Pull request merged**